Mathlib/SetTheory/Ordinal/CantorNormalForm.lean

```diff
@@ -65,80 +63,132 @@ We special-case `CNF 0 o = CNF 1 o = [(0, o)]` for `o ≠ 0`.
 `CNF b (b ^ u₁ * v₁ + b ^ u₂ * v₂) = [(u₁, v₁), (u₂, v₂)]` -/
 @[pp_nodot]
 def CNF (b o : Ordinal) : List (Ordinal × Ordinal) :=
-  CNFRec b [] (fun o _ho IH ↦ (log b o, o / b ^ log b o)::IH) o
+  CNFRec b [] (fun o _ IH ↦ (log b o, o / b ^ log b o)::IH) o
+
+/-- The exponents of the Cantor normal form are stored in the first entries. -/
+def CNF.exponents (b o : Ordinal) := (CNF b o).map Prod.fst
+/-- The coefficients of the Cantor normal form are stored in the second entries. -/
+def CNF.coeffs (b o : Ordinal) := (CNF b o).map Prod.snd
 
```

Comment:
I think this wording might be a bit more confusing, since the Cantor normal form in its standard mathematical description has no such thing as first or second entries. Maybe this could read
```
The exponents of the Cantor normal form are stored in the first entries of elements in `CNF`.
```

Reply:
Sounds good!